ext/civigrant/templates/CRM/Grant/Form/Selector.tpl

```diff
@@ -33,7 +33,7 @@
 
   {counter start=0 skip=1 print=false}
   {foreach from=$rows item=row}
-  <tr id='crm-grant_{$row.grant_id}' class="{cycle values="odd-row,even-row"} crm-grant crm-grant_status-{$row.grant_status_id}">
+  <tr id='crm-grant_{$row.grant_id}' class="{cycle values="odd-row,even-row"} crm-grant crm-grant_status-{$row.grant_status}">
```

Comment:
note that I think we want the label not id here but if we wanted id then we would need to modify the default return properties in CRM_Grant_BAO_Query